oc-game-swift/Game/Game.swift

```diff
@@ -7,3 +7,106 @@
 //
 
 import Foundation
+
+// Create the Game class
+class Game {
+    static var playerList = [Player]()
```

Comment:
it is better to declare type and define a default value : 
```swift
static var playerList: [Player] = []

Reply:
Done